src/app/modules/time-entries/pages/time-entries.component.html

```diff
@@ -64,13 +64,17 @@
     <div class="modal-content" cdkDrag (cdkDragEnded)="resetDraggablePosition($event)">
       <div class="modal-header" cdkDragHandle>
         <h5 class="modal-title">{{ entryId ? 'Edit Entry' : 'New Entry' }}</h5>
+        <button type="button" class="btn" (click)="detailsFields.cleanFormWithSkipProject()">
```

Comment:
How work with  #Id **detailsFields**?
I think this should be this.cleanFormWithSkipProject()

Reply:
With the `#detailFields` a local template variable associated with the ` app-details-fields ` component is created, this allows access to the functions and data of the ` app-details-fields ` component (child component) in this case when you click on the icon, the function `cleanFormWithSkipProject()` that is in the `details-fields.component.ts` is executed as it is not a function of the component `time-entries.component.ts` can not be accessed with this, but otherwise `this.detailsFields.cleanFormWithSkipProject() ` can be used but i think its not neccesary.